import-service/serverless.yml

```diff
@@ -0,0 +1,60 @@
+# For full config options, check the docs:
+#    docs.serverless.com
+service: import-service
+
+plugins:
+    - serverless-plugin-typescript
+
+configValidationMode: warn
+
+custom:
+  s3BucketName: maksim-charnou-products-files
+
+provider:
+    name: aws
+    runtime: nodejs16.x
+    stage: 'dev'
+    region: eu-west-1
+    profile: aws
+    iam:
+        role: 
+            statements:
+                - Effect: 'Allow'
+                  Action:
+                    - 's3:ListBucket'
+                  Resource:
+                    - "arn:aws:s3:::${self:custom.s3BucketName}"
+                - Effect: Allow
+                  Action: "s3:*"
+                  Resource:
+                    - "arn:aws:s3:::${self:custom.s3BucketName}/*"
+    environment:
+        PG_HOST: awsdb.c2cf02vn1xvw.eu-west-1.rds.amazonaws.com
+        PG_PORT: 5432
+        PG_DB_NAME: awsdb
+        PG_USERNAME: postgres
+        PG_PASSWORD: 0dwrJFVYpcjXCKBMpccN
```

Comment:
the better approach would be to store all this creds in '.env' file which should not be stored in repo. The current way is insecure